plugin.json

```diff
@@ -59,6 +59,20 @@
                 "help_text": "The webhook secret used by the ServiceNow API calls to Mattermost for sending message response.",
                 "regenerate_help_text": "Regenerates the secret for ServiceNow Virtual Agent Plugin. Regenerating this key invalidates any existing key.",
                 "default": ""
+            },
+            {
+                "key": "ChannelCacheSize",
+                "display_name": "DM Channel Cache Size:",
+                "type": "number",
+                "help_text": "The size of the cache that is used to store DM channel IDs.",
+                "default": 10000
+            },
+            {
+                "key": "ChannelCacheTTL",
+                "display_name": "DM Channel Cache Expiry Time:",
+                "type": "number",
+                "help_text": "The expiry time (in minutes) after which the entries in the cache are deleted.",
+                "default": 10
```

Comment:
That seems quite low. Fetching all channels every 10 minutes is quite often. Maybe a day?